ksql-functional-tests/src/test/java/io/confluent/ksql/test/TopologyFileRewriter.java

```diff
@@ -62,10 +63,11 @@
 
   private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
-  private TopologyFileRewriter() {
+  public TopologyFileRewriter() {
   }
 
-  public static void main(final String[] args) throws Exception {
+  @Test
```

Comment:
Now runs as a test so that is has the correct class path. If run from `main` it won't find the test UDFs and fails.